changeBackGroundColor.js

```diff
@@ -0,0 +1,26 @@
+function getRandomColor() {
+  var digits = "0123456789ABCDEF";
+  var color = '#';
+  for (var i = 0; i < 6; i++) {
+    color += digits[Math.floor(Math.random() * 16)];
+  }
+  return color;
+}
+
+var usedColorsArray = [];
+
+function changeBackground() {
+  var newColor = getRandomColor();
+  while (usedColorsArray.indexOf(newColor) >= 0) {
+    newColor = getRandomColor();
+  }
+  document.body.style.backgroundColor = newColor;
+  document.body.style.transition = "all 1s ease-in-out";
+  usedColorsArray.push(newColor);
+}
+
+for (var i = 0; i < 10; i++) {
```

Comment:
this approach dosnt seem to be flexible, what if we want to change the specs to change background color every .55 seconds for 10.21 seconds?